functions/Set-DbaTempDbConfig.ps1

```diff
@@ -201,15 +201,16 @@ function Set-DbaTempDbConfig {
                 }
             }
 
-            if (-not $LogFileSize) {
-                $LogFileSize = [Math]::Floor($DataFileSize / 4)
-            }
+            $logfile = $server.Databases['tempdb'].ExecuteWithResults("SELECT name, physical_name as FileName, size/128 as Size FROM sys.database_files WHERE file_id = 2").Tables[0].Rows[0];
```

Comment:
```suggestion
            $logfile = Get-DbaDbFile -SqlInstance $server -Database tempdb | Where-Object Type -eq 1 | Select-Object LogicalName, PhysicalName, @{L="SizeMb";E={$_.Size.Megabyte}}
```

Reply:
Thanks Shawn! I used the function `$server.Databases['tempdb'].ExecuteWithResults ()` because it is used in lines 151, 162, 174, 175, 187 to get similar information. So, I decided to use the same way to make the script homogeneous. I think that the purouse of that is to use only one session during the whole execution.
Anyway, if we decide to use the function you recommend in the lines I changed, Shouldn't we use it in all those lines too?

Reply:
You are still using the same session with the method provided.

Reply:
I have not touched this command but yes if you find the ExcuteWithResults used it can be replaced.

We also have a cleaner method for executing a query that you can find used in other commands. Our dbatools library includes a alias to that as just `Query()` so you can call `$server.Databases.Query("your query","database name")` and that returns the object as output directly.

Reply:
Ok, thanks. I'll use the recommended method in the line I added and I'll replace the function ExcuteWithResult by other clearer ways in the rest of the lines.